src/widgets/ReportTable/ReportCell.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import { APP_FONT_FAMILY, BACKGROUND_COLOR } from '../../globalStyles';
+
+export const ReportCell = ({ even, children }) => {
```

Comment:
> It's a little more intuitive imo to have a prop `value` so instead, we just do `<ReportCell value={"Some content"}/>` and in the return we return `{value}` rather than `{children}`

I like that idea you just mentioned! and it looks clearer btw. Agree, better I keep it like this now and move forward, then we can work on this important stuff. :-)